Makefile

```diff
@@ -10,8 +10,11 @@ ARCH := $(shell uname -m)
 # architectures as amd64
 ifeq ($(ARCH), x86_64)
 	OS_ARCH=darwin_amd64
+else ifeq ($(ARCH), arm64)
```

Comment:
It looks like you made this change in the `Recreate missing resources` commit 😄

Reply:
Hah! Oops.